app/views/cases/foi/_case_details.html.slim

```diff
@@ -1,10 +1,10 @@
 .grid-row
-  .column-full.case-details
+  - link_type_prefix = defined?(link_type) && link_type.present? ? "#{link_type}-" : ""
```

Comment:
This could perhaps be refactored out into a helper method as we're dong the same thing in at least one other place (not that I'm a fan of helper methods - but this seems a good case for one).

Reply:
Ya, good call, worth doing.